### DISCUSSION TITLE
DIVERSE ASSETS (2D/PIXEL/3D)

### DISCUSSION THREAD

**Discussion by Lunaire333** [AmyE123/TactiKit#9] [Map Editor Ideas]
Including loads of assets from 2D to 3D, including textures and so on, would be a great plus for the designers/artists with a vast imagination and a crazy need to make something out of this world. 

It would make the experience more enjoyable and activate their train of thoughts to generate more ideas.

![image](https://github.com/AmyE123/TactiKit/assets/158181584/d0ffb415-462e-491f-8dd0-370296099e3b)

**Comment by AmyE123** [comment]
Shouldn't be hard to set up something like this! I'm focusing on 3D mostly, but making the tiles into sprites won't be much work!
